client/src/utils/helpers.ts

```diff
@@ -0,0 +1,11 @@
+export function openURL(url: string) {
+  const canOpenNewTab = window.open(url, '_blank')
+
+  // if the browser blocks the new tab, open the url in the current tab
+  // this is the case for Safari on iOS
+  if (!canOpenNewTab) {
+    let a = document.createElement('a')
+    a.href = url
+    a.click()
```

Comment:
I didn't test it but couldn't we just do `window.location.href = url`? Or is it not gonna work for Safari either?

Reply:
Yes, works fine with your suggestion, I changed it. This code was left after trying some test to open a new tab in safari for iOS